website/versioned_docs/version-0.5.x/examples.mdx

```diff
@@ -0,0 +1,21 @@
+# Starter Applications and Tutorials
+Looking for code samples, starter applications, and tutorials to kickstart your development process? Check out some of the examples below for inspiration. 
+
+## Starter Applications
+- [**Social App**](https://github.com/ceramicstudio/ComposeDbExampleApp) - A starter application built around a social media platform use-case (this application utilizes the same data models used by the [ComposeDB Sandbox](../../sandbox))
```

Comment:
Change "Social App" to "Official ComposeDB Example App"?

Reply:
Done!